src/library/dlgcoverartfullsize.cpp

```diff
@@ -11,10 +11,9 @@
 #include "track/track.h"
 #include "util/widgethelper.h"
 
-DlgCoverArtFullSize::DlgCoverArtFullSize(QWidget* parent, BaseTrackPlayer* pPlayer)
-        : QDialog(parent),
-          m_pPlayer(pPlayer),
-          m_pCoverMenu(make_parented<WCoverArtMenu>(this)) {
+DlgCoverArtFullSize::DlgCoverArtFullSize(
+        QWidget* parent, BaseTrackPlayer* pPlayer, QMenu* pCoverMenu)
+        : QDialog(parent), m_pPlayer(pPlayer), m_pCoverMenu(pCoverMenu) {
```

Comment:
We use the stacked style of initialization list. Please revert to the style change (and help clang-format)